@filter-copy-to is not processed by maplinkimpl.xsl

## Expected Behavior
The filter-copy-to file should be included in the maplinkimpl processing. 

## Actual Behavior
Related Information <related-links> was not injected as the target file in @filter-copy-to is not included.

## Possible Solution
In maplinkimpl file, under `<!-- Match a topicref. Create all of the hierarchy links associated with the topicref. -->
<xsl:when test="@copy-to and (not(@format) or @format = 'dita') and not(contains(@chunk, 'to-content'))">`
It should include also when test @filter-copy-to.  The current code only cater for when test @copy-to.

## Steps to Reproduce
1. a map with a ditavalref.
2. this map includes 2 topicref with the same target topic file.
3. The topicref was keyref in the reltable with another topic for related information.
4. generate pdf output.

## Copy of the error message, log file or stack trace
...
[preprocess2.topic-fragment] Processing file:/D:/IxiaSoft/OutputGenerator/development/temp/PDF.tiewhong.76.1660533361141/temp/7e516ec33ec712bde45945912a3f213628b79cd1.xml
[preprocess2.topic-fragment] Processing file:/D:/IxiaSoft/OutputGenerator/development/temp/PDF.tiewhong.76.1660533361141/temp/**7e516ec33ec712bde45945912a3f213628b79cd1-1.xml**
...

topic-maplink:
  [maplink] Processing file:/D:/IxiaSoft/OutputGenerator/development/temp/PDF.tiewhong.76.1660533361141/temp/7e516ec33ec712bde45945912a3f213628b79cd1.xml

The 7e516ec33ec712bde45945912a3f213628b79cd1-1.xml should be also processed by maplink but it isn't.


## Environment
<!-- Include relevant details about the environment you experienced this in. -->

* DITA-OT version: 3.6, 3.7.2
* How did you run DITA-OT?
dita command
* Transformation type:
pdf

@hongyung - when I run your testcase, I get some errors:

```
 [gen-list] [DOTJ013E][ERROR] Failed to parse the referenced file 'file:/mnt/c/nobackup/3989/mpq1660122400946.ditamap'.: file:/mnt/c/nobackup/3989/mpq1660122400946.ditamap Line 5:Attribute "ixia_locid" must be declared for element type "topicref".
   [mapref] file:/mnt/c/nobackup/3989/vfl1660124975352.ditamap:6:83: [DOTX031E][ERROR]: The file mpq1660122400946.ditamap is not available to resolve link information.
   [mapref] file:/mnt/c/nobackup/3989/vfl1660124975352.ditamap:6:83: [DOTX031E][ERROR]: The file mpq1660122400946.ditamap is not available to resolve link information.
   [mapref] file:/mnt/c/nobackup/3989/vfl1660124975352.ditamap:6:83: [DOTX031E][ERROR]: The file mpq1660122400946.ditamap is not available to resolve link information.
```

and in the PDF output, there is no "Topic A", and there is only one "Some test topic" and it does have a "Related information" section. Can you please provide a testcase that reproduces the issue with a plain DITA-OT installation?

--- a/src/main/plugins/org.dita.base/xsl/preprocess/maplinkImpl.xsl
+++ b/src/main/plugins/org.dita.base/xsl/preprocess/maplinkImpl.xsl
@@ -95,6 +95,9 @@ See the accompanying LICENSE file for applicable license.
         <xsl:when test="@copy-to and (not(@format) or @format = 'dita') and not(contains(@chunk, 'to-content'))">
           <xsl:value-of select="dita-ot:normalize-uri(@copy-to)"/>
         </xsl:when>
+        <xsl:when test="@filter-copy-to and (not(@format) or @format = 'dita') and not(contains(@chunk, 'to-content'))">
+          <xsl:value-of select="dita-ot:normalize-uri(@filter-copy-to)"/>
+        </xsl:when>
         <xsl:otherwise>
           <xsl:value-of select="dita-ot:normalize-uri(@href)"/>
         </xsl:otherwise>

